doc/en/apiref.rst

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
@RonnyPfannschmidt  
yes, you are correct, I used git mv only but after that changed particular text in file as well.